blocks/api/children.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import { castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { renderToString } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import * as node from './node';
```

Comment:
Seems like there's a cyclic dependency between node and children, Should we just merge them in the same file?

Reply:
>Seems like there's a cyclic dependency between node and children, Should we just merge them in the same file?

Well-spotted. There is, though Webpack seems to handle it without issue. I personally found some elegance in having independent files with overlapping APIs (`matcher`, `fromDOM`, `toHTML`) mapping one-to-one with the offered source type. I don't feel too strongly about it.